source/com.google.crypto.tink/tink-android/Transforms/Metadata.xml

```diff
@@ -0,0 +1,5 @@
+<metadata>
+    <remove-node path="/api/package[@name='com.google.crypto.tink']" />
```

Comment:
Why removing top level namespace. It could contain some important types...

I wasn't able to build your PR, so I rebased it on top of main, but still have some issues.

JPobst started new PR with simpler approach. We'll see

Reply:
Looks good to me, let's wait for the other PR👍